lambda-notes/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import { Route } from 'react-router-dom';
+
+import NavBar from "./Components/NavBar";
+
+import './App.css';
+import { notEqual } from 'assert';
+
+//TODO: 
```

Comment:
Some of these things seem pretty familiar! Remember a lot of this app is close to what you've built in the past, in terms of mapping through data and displaying a list, or linking to different pages in a SPA. Don't be afraid to self-plagiarize if it helps you! The key difference will be in how you work with the responses from the API, but this isn't as scary as it seems; you can use multiple calls to get the data you need, or look into lifecycle methods like componentDidUpdate(). And let me know if I can help you!